### ISSUE TITLE
Syntax Errors in Dev Console

### ISSUE THREAD

**Issue by sci4me** [ShahanaFarooqui/RTL#55]
I am getting the following errors in Chrome's developer console when trying to access RTL:
```
Uncaught SyntaxError: Unexpected token <runtime.ec2944dd8b20ec099bf3.js:1>
Uncaught SyntaxError: Unexpected token <polyfills.418928a701f2040ada02.js:1>
Uncaught SyntaxError: Unexpected token <main.b42b628ca80a5ffd43ce.js:1>
```

To reproduce:
```
git clone <repo-url>
cd <repo>
npm install
<edit config file> (mv sample-RTL.conf RTL.conf)
node rtl
```
I am using Node 10.15.1

**Comment by ShahanaFarooqui** [comment]
The code is repo is currently unstable as we are testing some changes for nginx and docker enablement. Please get the stable release version 0.2.0 and let us know if the issue still persists.

**Comment by sci4me** [comment]
@ShahanaFarooqui That fixed it, thanks! Suggestion: might be nice to put the latest usable version in the README or something. :)

**Comment by ShahanaFarooqui** [comment]
@sci4me Thanks for the suggestion. We added the stable version detail in README. Our recent commit (2 mins back) fixes the master branch as well. The master branch code can be used with 'reverseProxy' configurations. If the application is used with nginx reverse proxy then, the reverseProxy setting should be 1 else 0. Please check readme link for more details on this. 'https://github.com/ShahanaFarooqui/RTL/blob/master/Application_configurations'

**Comment by ShahanaFarooqui** [comment]
@sci4me We released another stable version 0.2.8 today with final BTCPayserver dockerization changes. Please note that we decided to remove reverse_proxy from configuration as it is not required anymore. All the other settings will work as before.

**Comment by sci4me** [comment]
@ShahanaFarooqui Awesome! Thanks for keeping me in the loop!
